src/bundle/Resources/public/js/CKEditor/custom-attributes/custom-attributes-command.js

```diff
@@ -1,13 +1,47 @@
 import Command from '@ckeditor/ckeditor5-core/src/command';
 
 class IbexaCustomAttributesCommand extends Command {
+    cleanAttributes(modelElement, attributes) {
+        Object.entries(attributes).forEach(([elementName, config]) => {
+            if (elementName === modelElement.name) {
+                return;
+            }
+
+            this.editor.model.change((writer) => {
+                Object.keys(config).forEach((name) => {
+                    if (attributes[modelElement.name]?.[name]) {
```

Comment:
What would you say on moving checking if `attributes[modelElement.name]` outside, to line 6 for example? And here only `if (attributes[modelElement.name][name]) {`

Reply:
no, it has to be that way. We cannot do early return if there are some attributes for this element because we need to clean attributes from other elements